### PR TITLE
parse pol uuid

### PR DESCRIPTION
**Pull request by rombernier**
context: https://github.com/SekoiaLab/integration/issues/150

### CODE REVIEW

File: Fortinet/fortigate/ingest/parser.yml [hunk 1]
```diff
@@ -240,7 +240,7 @@ stages:
           fortinet.fortigate.attack.name: "{{parsed_event.message.attack}}"
           fortinet.fortigate.attack.id: "{{parsed_event.message.attackid}}"
           fortinet.fortigate.policyid: "{{parsed_event.message.policyid}}"
-          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
+          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid or parsed_event.message.uuid}}"
```
Comment (squioc) [inline Fortinet/fortigate/ingest/parser.yml:243]: I wonder if we shouldn't move this assignment in a dedicated `set` action with a filter. The `uuid` field is too generic and may contain information that is not related to a policy. I suggest:

```suggestion
          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
          
      - set:
          fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
        filter: "{{parsed_event.message.cfgpath == 'firewall.policy'}}"
```

**Review by squioc — COMMENTED** [review]
Thank you for the contribution.
I wonder if the assignment of the custom field from the `uuid` field shouldn't be protected by a filter.

File: Fortinet/fortigate/ingest/parser.yml (hunk 1 above)
Comment (rombernier) [reply Fortinet/fortigate/ingest/parser.yml]: Yes, you're right, I made the change.

**Review by squioc — APPROVED** [review]
LGTM
